rest/change_point_routes.go

```diff
@@ -42,36 +43,56 @@ func (h *perfGetChangePointsByVersionHandler) Parse(_ context.Context, r *http.R
 	catcher := grip.NewBasicCatcher()
 	var err error
 	page := vals.Get("page")
+	delete(vals, "page")
 	if page != "" {
 		h.args.Page, err = strconv.Atoi(page)
 		catcher.Add(err)
 	} else {
 		h.args.Page = 0
 	}
 	pageSize := vals.Get("pageSize")
+	delete(vals, "pageSize")
 	if pageSize != "" {
 		h.args.PageSize, err = strconv.Atoi(pageSize)
 		catcher.Add(err)
 	} else {
 		h.args.PageSize = 10
 	}
 	h.args.VariantRegex = vals.Get("variantRegex")
+	delete(vals, "variantRegex")
 	h.args.VersionRegex = vals.Get("versionRegex")
+	delete(vals, "versionRegex")
 	h.args.TaskRegex = vals.Get("taskRegex")
+	delete(vals, "taskRegex")
 	h.args.TestRegex = vals.Get("testRegex")
+	delete(vals, "testRegex")
 	h.args.MeasurementRegex = vals.Get("measurementRegex")
-	tls := vals.Get("threadLevels")
-	if tls != "" {
-		tlslice := strings.Split(tls, ",")
-		for _, tl := range tlslice {
-			intTl, err := strconv.Atoi(tl)
-			catcher.Add(err)
-			h.args.ThreadLevels = append(h.args.ThreadLevels, intTl)
+	delete(vals, "measurementRegex")
+	h.args.Arguments = map[string][]int{}
```

Comment:
I have a similar sentiment here as my other comment. It worries me to re-type these, it seems an easy place to accidentally break in the future. What would you think about declaring these queryParams as constants or something like that so they could be re-used?